ts/src/binance.ts

```diff
@@ -4102,26 +4102,15 @@ export default class binance extends Exchange {
         const marketType = ('closePosition' in order) ? 'contract' : 'spot';
         const symbol = this.safeSymbol (marketId, market, undefined, marketType);
         const filled = this.safeString (order, 'executedQty', '0');
-        let timestamp = undefined;
-        let lastTradeTimestamp = undefined;
-        if ('time' in order) {
-            timestamp = this.safeInteger (order, 'time');
-        } else if ('workingTime' in order) {
-            lastTradeTimestamp = this.safeInteger (order, 'transactTime');
-            timestamp = this.safeInteger (order, 'workingTime');
-        } else if ('transactTime' in order) {
-            lastTradeTimestamp = this.safeInteger (order, 'transactTime');
-            timestamp = this.safeInteger (order, 'transactTime');
-        } else if ('createTime' in order) {
-            lastTradeTimestamp = this.safeInteger (order, 'updateTime');
-            timestamp = this.safeInteger (order, 'createTime');
-        } else if ('updateTime' in order) {
+        const timestamp = this.safeIntegerN (order, [ 'time', 'createTime', 'workingTime', 'transactTime', 'updateTime' ]); // order of the keys matters here
+        let lastTradeTimestamp = this.safeInteger (order, 'transactTime');
```

Comment:
btw, it doesnt seem ideal to me, that `transactTime` can be both order's timestamp and last update's timestamp at the same time.

Reply:
in the past I had a PR which added "lastUpdateTimestamp" in addition to "lastTradeTimestamp" because the former is a separate thing, i.e. when you edit-order (which can be "last update" timestamp) in many cases.

Reply:
@ttodua the problem is `transactTime` appears to have different meanings depending on the context (spot edit order or createOrder with { "newOrderRespType": "FULL" })